cves/CVE-2017-14056.yml

```diff
@@ -91,13 +101,9 @@ fixes_vcc_instructions: |
 fixes:
 - commit: 96f24d1bee7fe7bac08e2b7c74db1a046c9dc0de
```

Comment:
I know that notes are optional here but I think adding them for the fix and vccs would be helpful. I think it's worth noting that the vulnerability causing commit was from the creation of the file, which can indicate a lack of testing/revisions.